src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -131,7 +129,12 @@ const events = [
 			node.is_media_node() &&
 			(name === 'videoHeight' || name === 'videoWidth')
 	},
-
+	{
+		event_names: ['loadeddata', 'canplay', 'canplaythrough', 'emptied'],
```

Comment:
From where did you derive the event names that may change the `readyState`? Looking at https://html.spec.whatwg.org/multipage/media.html#ready-states I don't see `emptied` but I see `waiting` which is missing here (there's also `timeupdate` but we shouldn't use that as it's fired all the time)